dev-libs/nfd/nfd-1.0.recipe

```diff
@@ -0,0 +1,39 @@
+SUMMARY="Native File Dialog"
+DESCRIPTION="A tiny, neat C library that portably invokes native file open and \
+save dialogs."
+HOMEPAGE="www.frogtoss.com"
+COPYRIGHT="2014 Frogtoss Games"
+LICENSE="Zlib-generic"
+REVISION="1"
+SOURCE_URI="https://github.com/puckipedia/nativefiledialog/archive/39ecf6a1.zip"
+CHECKSUM_SHA256="6c4eff1c0d5f2d3165192710cb1c844ec9e63d4fe7c4e2238cf5c6ddcb0da648"
+SOURCE_DIR="nativefiledialog-39ecf6a1d8c8cfa3a6f1c6795765911245809833"
+
+ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
+
+PROVIDES="
+	nfd = $portVersion
+	lib:nfd = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc
+	cmd:scons
+	"
+
+BUILD()
+{
+	cd src/
+	scons debug=0
```

Comment:
Only the building with scons should be in BUILD(), the copying around should be in INSTALL()